src/test/electron.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+const path = require('path')
+const execa = require('execa')
+const { hook } = require('../utils')
+
+module.exports = (argv) => {
+  const input = argv._.slice(1)
+  const forwardOptions = argv['--'] ? argv['--'] : []
+  const watch = argv.watch ? ['--watch'] : []
+  const files = argv.files.length ? [...argv.files] : ['test/**/*.spec.js']
```

Comment:
Is `argv.files` always defined?

Reply:
We've got a [default value](https://github.com/ipfs/aegir/blob/6ca0ae34d4100801c1f7c62287eaf8a62c247b79/cmds/test.js#L37-L42) for it so it should be declared by yargs if it's not passed.